lib/HalModuleParser.js

```diff
@@ -23,9 +23,9 @@ var when = require('when');
 var pipeline = require('when/pipeline');
 var utilities = require('./utilities.js');
 var buffers = require('h5.buffers');
```

Comment:
This module is no longer used in this file. I'd also check if we even need this dependency at all, given the changes in this PR.

Reply:
You are right, no longer required. Removed the dependency.